tock/tock/templates/employees/user_detail.html

```diff
@@ -4,7 +4,7 @@
 {% endblock %}
 
 {% block content %}
-  <h1>Let's Tock about {{ object.user.first_name }} {{ object.user.last_name }}!</h1>
+  <h1>Let's Tock about {{ object.user.user_data.display_name }}!</h1>
```

Comment:
Here, unless I'm mistaken, `object` is our `user_data` object, so if we add a convenience method to it, then we could just use that: `object.display_name` or whatever. On the other hand, if we're OK with following the django user object (which user_data extends) then we could use django's built-in `get_full_name` like this: `object.user. get_full_name`. 

Right now, if I'm reading it right, I think you're going from user_data up to user, then back down to user_data again.

Reply:
hah, awesome